user/noto/Backend.rst

```diff
@@ -91,5 +91,25 @@ pointing accuracy. This is accomplished using the command::
 
 which uploads to the system the parameters relative to section number [sect] 
 (you can generally use 0, which exists for all the receivers, unless you need 
-to observe only with a different feed).   
+to observe only with a different feed).
 
+
+.. _EN_calmux-focus-selector:
+
+Calmux - calibration diode multiplexer
+======================================
+
+The Calmux is a multiplexer that allows to choose which backned is entitled to control the
+calibration diode on the frontend. This device is required when more than one backend has the
+capability to exploit the fast switching. In Noto the DBBC (for VLBI) and the TotalPower have such
+a feature. When observing with DISCOS the proper calmux configuration is done transparently during
```

Comment:
When observing with DISCOS, the proper calmux configuration is set transparently during the setup phase.